src/pipeline.ts

```diff
@@ -482,14 +523,400 @@ export async function scanRssFeeds() {
 		});
 		await indexNewTorrents();
 		for (const [i, candidate] of candidatesSinceLastTime.entries()) {
+			const candidateLog = `${chalk.bold.white(candidate.name)} from ${candidate.tracker}`;
 			logger.verbose({
 				label: Label.RSS,
-				message: `Processing release ${i + 1}/${
-					candidatesSinceLastTime.length
-				}`,
+				message: `(${i + 1}/${candidatesSinceLastTime.length}) ${candidateLog}`,
 			});
-			await checkNewCandidateMatch(candidate, Label.RSS);
+			await checkNewCandidateMatch(candidate, candidateLog, Label.RSS);
 		}
 		logger.info({ label: Label.RSS, message: "Scan complete" });
 	}
 }
+
+export async function injectSavedTorrents() {
```

Comment:
this is a beast of a function—I think it'd be good to break up into smaller pieces. Perhaps in a different file as well, though I'm not sure what the good division lines would be.